apis/install/v1alpha1/common.go

```diff
@@ -21,6 +21,7 @@ import (
 )
 
 type Image struct {
```

Comment:
I don’t think you need this.

Reply:
I think I am wrong.  I notice that executor and binoculars don't use Image but we probably do want that.  Don't change this after all.

Reply:
removing -- "repository:tag" are the only real naming attributes